bot/utils/extensions.py

```diff
@@ -31,4 +33,12 @@ def on_error(name: str) -> NoReturn:
         yield module.name
 
 
+async def invoke_help_command(ctx: Context, *commands: str) -> None:
```

Comment:
You can receive full name with parents using `ctx.command.qualified_name`, so you don't have to pass command name manually.